spec/index.bs

```diff
@@ -1391,7 +1390,7 @@ internal slots of a [=texture=] internal object once we have one. -->
     If 0, defaults to |texture|.{{GPUTextureDescriptor/mipLevelCount}} &minus; {{GPUTextureViewDescriptor/baseMipLevel}}.
 
   * {{GPUTextureViewDescriptor/arrayLayerCount}}:
-    If 0, defaults to |texture|.{{GPUTextureDescriptor/arrayLayerCount}} &minus; {{GPUTextureViewDescriptor/baseArrayLayer}}.
+    If 0, defaults to |texture|.{{GPUTextureDescriptor/size}}.[=Extent3D/depth=] &minus; {{GPUTextureViewDescriptor/baseArrayLayer}}.
```

Comment:
This can be confusing for the cube arrays. We can't say that the view array count defaults to the depth in this case.

Reply:
Resolved in meeting, this is an orthogonal issue and we'll open a new issue.